DeepSpeech.py

```diff
@@ -670,12 +671,77 @@ def __call__(self, progress, data, **kwargs):
         log_info('FINISHED optimization in {}'.format(datetime.utcnow() - train_start_time))
     log_debug('Session closed.')
 
+def finetune_lm_params():
```

Comment:
Please move this into a separate file, similar to how `evaluate.py` is structured.